lib/git_reflow/commands/refresh.rb

```diff
@@ -0,0 +1,24 @@
+desc 'refresh will rebase your current branch based on the base branch'
+long_desc <<LONGTIME
+  Performs the following:\n
+  \t$ git checkout <base_branch>\n
+  \t$ git pull <remote_location> <base_branch>\n
+  \t$ git checkout <current_branch>\n
+  \t$ git rebase <base_branch>\n
+LONGTIME
+arg_name '[base_branch] - branch that you want to rebase on (master by default), [remote_location] - remote repository name to fetch updates from (origin by default)'
+command :refresh do |c|
+  c.action do |global_options, options, args|
+    if args.empty?
+      raise "usage: git reflow refresh [base_branch_name - defaults to master] [remote_location - defaults to origin]"
+    else
+      # defaults remote_location to origin
+      remote_location = args.length < 2 ? "origin" : args[1]
```

Comment:
I think the `remote` arg should come first to match that of `git pull`. I know I would mix these up when running it otherwise.